CONTRIBUTING.md

```diff
@@ -48,6 +48,34 @@ Use proper judgement, keeping simplicity and user-friendliness as the top priori
 
 When in doubt, have a look at a few existing pages :).
 
+## Directory structure
+
+The English pages directory is `pages`, under which are the platform directories. Language specific directories must follow the pattern `pages.<locale>`, where `<locale>` is a [POSIX Locale Name](https://www.gnu.org/software/gettext/manual/html_node/Locale-Names.html#Locale-Names) in the form of `<language>[_<country>]`, where:
+
+- `<language>` is the shortest [ISO 639](https://en.wikipedia.org/wiki/ISO_639) language code for the chosen language (see [here](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) for a complete list).
+- `<country>` is the two-letter [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) country code for the chosen region (see [here](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements) for a complete list).
+
+[!NOTE]
+> Do not put any page directly inside the English pages directory or language specific directories.
+
+The `<country>` code is optional and should only be added when it is needed. In other words, only when there is a valid reason to distinguish between a language (`ll`) and its regional dialects (`ll_CC1`, `ll_CC2`, etc.). For example, both `fr_FR` and `fr_BE` should fall under the same `pages.fr` directory since there virtually is no difference in writing between standard French and Belgian French.
```

Comment:
I think that we should avoid subjective language like "valid". Who decides what is valid and what is not? Maybe we could just omit this section. Afaik there has never been any confusion on which pages belong in which language directory. This is intuitively clear to most contributors.

Reply:
Okay, but I think that "valid" is not an ideal choice of words. For example, swiss german differs in writing from standard german, both in choice of words and in the spelling of words. However, a case can be made for either including it in the pages.de directory or for creating a separate one. Saying that it would be invalid to create a separate directory is (in my opinion) invalidating swiss german's right to exist. 

Also, I disagree that there shouldn't be separate directories for fr and fr_BE.

Reply:
How about "only when three is a substancial difference between a language..."?

As for fr vs. fr_BE, I don't know if there is a huge difference or not. Like, the difference is just about a few not that common words or not? For example, there isn't a directory only for en_UK.

If we decide to split the dialects, we should replace this example.

Reply:
> How about "only when three is a substancial difference between a language..."?

I think that this would be a great phrasing!